multicluster/transport.go

```diff
@@ -52,6 +52,17 @@ func (op ForCluster) ApplyToTransport(t *Transport) {
 	t.cluster = pointer.String(string(op))
 }
 
+func asTransport(rt http.RoundTripper) (*Transport, bool) {
```

Comment:
Seems there is no need to return *Transport

Reply:
Here we return Transport so that then NewTransportWrapper can enforce options in the inside wrapped Transport.